apps/drupal/particle.info.yml

```diff
@@ -5,6 +5,9 @@ base theme: stable
 core: 8.x
 libraries:
   - particle/core
+libraries-override:
```

Comment:
Need a note here about what we're doing and the potential ramifications (like jQuery compatibility, lodash/underscore compatibility).